### PR TITLE
Add map card documentation.

### PR DESCRIPTION
**Pull request by jeradM**
**Description:**
Adds documentation for the lovelace map card added in 0.73.1

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_lovelace/map.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+sidebar_label: Map
+description: "A card that allows you to display entities on a map"
+date: 2018-07-08 10:28 +00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+### Map
```
Comment (balloob) [inline source/_lovelace/map.markdown]: No need for a title.

**Review by balloob — DISMISSED** [review]

File: source/_lovelace/map.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+A card that allows you to display entities on a map
+
+<p class='img'>
+<img src='/images/lovelace/lovelace_map_card.png' alt='Screenshot of the map card'>
+Screenshot of the map card.
+</p>
+
+**Options**
```
Comment (sjabby) [inline source/_lovelace/map.markdown]: Should be linkable?

File: source/_lovelace/map.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| entity | string | **Required** | An entity_id. Example: 'device_tracker.demo_paulus'.
+
+<p class='note'>
+  Only entities that have latitude and longitude attributes will be displayed on the map
+</p>
+
+
+**Examples**
```
Comment (sjabby) [inline source/_lovelace/map.markdown]: Should be linkable?

File: source/_lovelace/map.markdown [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+**Options**
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| type | string | **Required** | `map`
+| entities | list | **Required** | A list of entities to display on the map. Can be entity_ids or entity objects (see structure below).
+| title | string | Optional | Card title
+| aspect_ratio | string | `"100%"` | Map height:width ratio
+
```
Comment (sjabby) [inline source/_lovelace/map.markdown]:
```
{% configuration %}
type:
  required: true
  description: map
  type: string
cards:
  required: true
  description: A list of entities to display on the map. Can be entity_ids or entity objects (see structure below).
  type: list
title:
  required: false
  description: Card title
  type: string
aspect_ratio:
  required: false
  description: Map height:width ratio
  type: string
{% endconfiguration %}
```

File: source/_lovelace/map.markdown [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+`entity` object
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| entity | string | **Required** | An entity_id. Example: 'device_tracker.demo_paulus'.
+
```
Comment (sjabby) [inline source/_lovelace/map.markdown]:
```
{% configuration %}
entity:
  required: true
  description: "An entity_id. Example: 'device_tracker.demo_paulus'."
  type: string
{% endconfiguration %}
```

**Review by DubhAd — APPROVED** [review]
👍

**Pull request merged**